src/main/java/org/symphonyoss/symphony/messageml/util/IUserPresentation.java

```diff
@@ -28,4 +28,5 @@
   String getScreenName();
   String getPrettyName();
   String getEmail();
+  String getInteractionUrl();
```

Comment:
Open question: I'm wondering if _interactionUrl_ is really a part of _UserPresentation_. What does the property represent? Is it a URL for interacting with the user? Please add Javadocs to clarify.